2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +12,14 @@
   
 */
 
-function getSettlers() {}
+// const getSettlers = (colonisers) =>
+//   colonisers.filter((item) => item.slice(-6) == "family" && item[0] == "A");
+
+function getSettlers(colonisers) {
+  return colonisers.filter(
+    (item) => item.slice(-6) == "family" && item[0] == "A"
```

Comment:
You may also consider using **startsWith** and **endsWith** methods

Reply:
Thank you, I've not heard of these before but looking into them now, thanks!